add-chain/cmd/convert-to-toml.go

```diff
@@ -0,0 +1,41 @@
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+	"runtime"
+
+	"github.com/ethereum-optimism/superchain-registry/add-chain/config"
+	"github.com/ethereum-optimism/superchain-registry/add-chain/flags"
+	"github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/urfave/cli/v2"
+)
+
+var ConvertToTOMLCmd = cli.Command{
```

Comment:
This is a helper command to convert existing chain config .yaml files to .toml. Can be removed before this pr is merged

Reply:
Done here: https://github.com/ethereum-optimism/superchain-registry/pull/386/commits/347a835f491592b51b80bd6db63ec74a2554dfce